pkg/cassandra/datacenter.go

```diff
@@ -397,6 +397,16 @@ func AddContainersToPodTemplateSpec(dcConfig *DatacenterConfig, containers []cor
 		return errors.New("PodTemplateSpec was nil, cannot add containers")
 	} else {
 		dcConfig.PodTemplateSpec.Spec.Containers = append(dcConfig.PodTemplateSpec.Spec.Containers, containers...)
+		if dcConfig.ServerType == api.DistributionDse {
+			index, found := FindContainer(dcConfig.PodTemplateSpec, "cassandra")
+			if found {
+				env := dcConfig.PodTemplateSpec.Spec.Containers[index].Env
+				cassandraConfEnvVar := utils.FindEnvVar(env, "CASSANDRA_CONF")
+				if cassandraConfEnvVar == nil {
+					dcConfig.PodTemplateSpec.Spec.Containers[index].Env = append(dcConfig.PodTemplateSpec.Spec.Containers[index].Env, corev1.EnvVar{Name: "CASSANDRA_CONF", Value: "/config"})
+				}
+			}
+		}
```

Comment:
The images we're using right now aren't creating the appropriate symlinks for DSE to find the config files.
This hack will allow it to read the `cassandra.yaml` file in the right place, but not the `dse.yaml` file.

It will be enough to move forward with multi DC related work.